### DISCUSSION TITLE
Meta Tag for Unit Label

### DISCUSSION THREAD

**Discussion by Mark02420** [chuot/rdio-scanner#291] [Ideas]
Is it possible to add a meta tag for unit label (#UNITLBL) so that the unit name, if any, can be extracted from the file name?

**Comment by chuot** [comment]
I'll see if I can fit that in the upcoming v7

**Reply by cpg178-kcd** [reply]
Would it be possible to also look into including the ability to search by UID in the Search Call area along with adding UID to the info display on each stored call? I think a drop down would be way to large but if you knew a specific UID you could type it in and it would search for all calls from that ID

**Reply by Mark02420** [reply]
I agree.  The ability to search by unit ID and also show unit id and unit label in search results would be very helpful.

**Reply by chuot** [reply]
It's planned for v7.0, which should be released in early 2023. I'm working hard on it.
